pkgs/sdk-pkg/sdk_generator/output/github_triggers.json

```diff
@@ -3723,7 +3723,7 @@
         "interval": 10,
         "max_time": 180
       },
-      "url": "https://github.com/CiscoTestAutomation/genielibs/tree/master/pkgs/sdk-pkg/src/genie/libs/sdk/triggers/modify/bgp/iosxe/modify.py"
```

Comment:
These changes are automatically generated by `make json`, I have no idea why it does that